frontend/website/src/Render.re

```diff
@@ -195,6 +195,12 @@ Router.(
             <Wrapped> <Careers jobOpenings /> </Wrapped>
           </Page>,
         ),
+        File(
+          "testnet",
```

Comment:
instead of having it be a separate route, would it be possible to replace the current coda-testnet page in the docs?

Reply:
we need to fill this route to fix the SEO stuff and other links that already exist going to codaprotocol.com/testnet , but maybe we can do some sort of redirecty thing?

Reply:
For now I say we just link it from the docs but once we get the full version up we can redirect from the hello-coda page.